nginx/default.conf

```
@@ -37,7 +13,12 @@ set_real_ip_from   127.0.0.1;
 set_real_ip_from   10.0.0.0/8;
 set_real_ip_from   172.16.0.0/12;
 set_real_ip_from   192.168.0.0/16;
-
 real_ip_header     X-Forwarded-For;
-
 real_ip_recursive on;
+
+server {
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+	access_log /var/log/nginx/access.log logstash;
```

It can't have this name, some properties define the same format elsewhere, which would cause a clash when using their config with this image.

```suggestion
	access_log /var/log/nginx/access.log logstash_internal;
```